app/main/views/suppliers.py

```diff
@@ -12,13 +12,17 @@
 from dmapiclient.audit import AuditTypes
 from dmutils.email import send_email, generate_token, MandrillException
 from dmutils.documents import (
-    get_signed_url, get_agreement_document_path, file_is_pdf,
-    AGREEMENT_FILENAME, SIGNED_AGREEMENT_PREFIX, COUNTERSIGNED_AGREEMENT_FILENAME,
+    AGREEMENT_FILENAME, COUNTERSIGNED_AGREEMENT_FILENAME,
+    file_is_pdf, get_document_path, get_extension, get_signed_url,
+    generate_timestamped_document_upload_path, degenerate_document_path_and_return_doc_name
 )
 from dmutils import s3
 from dmutils.formats import datetimeformat
 
 
+FRAMEWORKS_IN_SUPPLIERS_PAGE = ['g-cloud-7', 'digital-outcomes-and-specialists']
```

Comment:
I think the plan is to not extend it. There should be no need for frameworks that follow the new flow to have the ability to download an individual signed agreement or upload an individual countersigned agreement.